conftest.py

```diff
@@ -250,3 +270,27 @@ def custom_product(randomizer, custom_location):
     product.company_id = custom_location.company_id
     product.location_id = custom_location.id
     return product
+
+
+@pytest.fixture(scope='function')
+def custom_products_for_filtering(randomizer, custom_location):
+    counter = 0
+    products = []
+    while counter < 3:
+        counter += 1
+        data = randomizer.product_data()
+        product = Product.objects.create(
+            company=custom_location.company,
+            name=data['name'],
+            logo=Image.objects.create(**data['logo']),
+            description=data['description'],
+            volume=data['volume'],
+            measure=data['measure'],
+            cost=data['cost']
+        )
+        product.locations.set([custom_location])
+        product.company_id = custom_location.company_id
+        product.location_id = custom_location.id
```

Comment:
to `create`

Reply:
can't delete product.location_id = custom_location.id 
because can not understand how to take value fron many to many field